src/lib/objects/index.d.ts

```diff
@@ -9,11 +10,15 @@ export interface WakuObjectArgs {
 	updateStore: (updater: (state: unknown) => unknown) => void
 
 	send: (data: unknown) => Promise<void>
+	sendTransaction?: (to: string, token: Token, fee: Token) => Promise<string>
+	estimateTransaction?: (to: string, token: Token) => Promise<Token>
+	onViewChange?: (view: string) => void
 }
 
 interface WakuObjectDescriptor {
 	readonly objectId: string
 	readonly wakuObject: ComponentType
+	readonly standalone?: ComponentType
```

Comment:
I assume this is what is shown when the object is being set up.

If yes, I would like to find a better name for it later. I think there will be three different major views (components) for an object:
1. when you set it up
2. chat widget
3. info screen

From the current designs 1. and 2. can be internally represented with other views (components).

Reply:
yes that is the case. Also it can be used to show `3. info screen` but likely that should be a different component.